internal_resources/contributing.adoc

```diff
@@ -3,6 +3,8 @@
 
 This guide will help new contributors use `git` and GitHub for Keycloak documentation submissions and suggestions.
 
+Before you start, please take a look at our writing templates contained in the link:https://github.com/keycloak/keycloak-documentation/tree/master/internal_resources[internal resources directory], which also describes each template file with information about when to use which. We are working to update our documentation to match the style of these templates and request that new contributions please use the format and style in them.
+
```

Comment:
Matthew, let's tweak  that last sentence say something like:

"We are working to modularize our documentation to separate procedural content, conceptual content, and reference content into separate .adoc files, and we kindly request that new contributions please use these templates." 
I know it is the same meaning, but it is worded more strongly. Thanks for adding the link!

Reply:
That is a reasonable suggestion, @tanberry. Thank you.